addons/website_slides/static/src/slide_category_one2many_field.js

```diff
@@ -14,4 +14,5 @@ SlideCategoryOneToManyField.defaultProps = {
     editable: "bottom",
 };
 
+SlideCategoryOneToManyField.additionalClasses = ['o_field_one2many'];
```

Comment:
picky std (style of the file)
```suggestion
SlideCategoryOneToManyField.additionalClasses = ["o_field_one2many"];